package.json

```diff
@@ -17,7 +17,7 @@
     "format": "prettier --write .",
     "prepare": "husky install",
     "migrate:create": "knex migrate:make",
-    "migrate:up": "knex migrate:latest",
+    "migrate:up": "NODE_OPTIONS='--import tsx/esm' knex migrate:latest",
```

Comment:
ESM problems :( though the good thing with this is that we can use this for the dev script too